src/strongbox/ui/cli.clj

```diff
@@ -574,18 +574,27 @@
              ;; failed to find or expand summary, probably because of selected game track.
              nil)))
 
+(defn-spec refresh-user-catalogue-item nil?
+  "re-fetch an item in the user catalogue using it's URI, replacing the old entry with any updated details. "
+  [addon :addon/summary]
+  (logging/with-addon addon
+    (info "refreshing details")
+    (let [dry-run? false
+          refreshed-addon (find-addon (:url addon) dry-run?)]
+      (if refreshed-addon
```

Comment:
this is an improvement to what was happening but uncaught errors will still fuck us. wrap this in a try ... catch

Reply:
also, test